### ISSUE TITLE
It does not work in java

### ISSUE THREAD

**Issue by naitz1337** [ddxoft/master#30]
I have tried your examples, I have not found a way to run it in java. Can you make a new example for Java?

**Comment by ddxoft** [comment]
download JNA library.
are you 64bit java?

Naitz ***@***.***> 于2022年1月19日周三 00:53写道：

**Comment by naitz1337** [comment]
yes i have tried windows 10, 64 bit java and everything, it does not give an error message. Does nothing. i need your help

**Comment by ddxoft** [comment]
DD.INSTANCE.DD_btn(0);//to add a line
[image: QQ截图20220119132740.png]

Naitz ***@***.***> 于2022年1月19日周三 02:57写道：

**Comment by naitz1337** [comment]
i tried it in java, c# it doesnt work. only c++ and python works, can you make a new example?
i can give teamviewer-anydesk

**Comment by naitz1337** [comment]
????????

**Comment by ddxoft** [comment]
???

Naitz ***@***.***> 于2022年1月20日周四 01:13写道：

**Comment by naitz1337** [comment]
it doesn't work java and c# can you make a new example? please

**Comment by ddxoft** [comment]
Naitz ***@***.***> 于2022年1月21日周五 07:24写道：
